.github/workflows/translations-sync.yml

```diff
@@ -23,7 +23,9 @@ jobs:
           config: 'crowdin.yml'
           upload_sources: true
           upload_translations: true
+          auto_approve_imported: true
           download_translations: true
+          skip_untranslated_strings: true
```

Comment:
I've added these 2 to speed up our translation workflow. It's annoying that when we change the keys the translations become unapproved. I think it's ok to consider translations added through GitHub as approved.

Reply:
Also, if we agree that translations should go only through Crowdin, there's no point in downloading untranslated strings, it's just polluting our JSON files and it's more of a mess when we want to change something